pennylane/resource/error.py

```diff
@@ -50,3 +52,22 @@ def get_error(approximate_op, exact_op, **kwargs):
             approximation.
         """
         raise NotImplementedError
+
+
+class ErrorOperation(Operation):
+    r"""Base class that represents quantum gates or channels applied to quantum
+    states and stores the error of the quantum gate.
+
+    .. note::
+        Child classes must implement the :func:`~.ErrorOperation.error` method which computes
+        the error of the operation.
+    """
+
+    @property
```

Comment:
The PRD predicts this to be an @abstractmethod and not a @property. Could you please elaborate on the change?

Reply:
yes indeed! Nice catch

Reply:
So the idea now it have it as both a `property` and an `abstractmethod`, right?

Reply:
Yes 👍🏼